ajax/libs/jquery.x18n/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.x18n",
+  "version": "1.1.3",
+  "description": "The jQuery adapter for the x18n internationalisiation library.",
+  "license": "MIT",
+  "filename": "jquery.x18n.min.js",
+  "keywords": [
+    "jquery",
+    "x18",
+    "javascript",
+    "internationalisation"
```

Comment:
I think we missed one important keyword?
https://github.com/florian/jQuery.x18n/blob/master/jQuery.x18n.jquery.json#L6
and maybe also `internationaliziation`?